package.json

```diff
@@ -19,5 +19,16 @@
   "bugs": {
     "url": "https://github.com/sass/sass-spec/issues"
   },
-  "homepage": "https://github.com/sass/sass-spec#readme"
+  "homepage": "https://github.com/sass/sass-spec#readme",
+  "dependencies": {
+    "js-yaml": "^3.14.0",
+    "node-hrx": "^0.1.0"
+  },
+  "devDependencies": {
+    "prettier": "^2.1.2",
+    "tap": "^14.10.8"
```

Comment:
`tap` is not even a dev dependency here, as it is used by the implementation of the spec runner